README.md

```diff
@@ -146,6 +149,24 @@ try {
 }
 ```
 
+## Custom error
+
+Developers can define their own custom error models to adapt to the specific requirements.
+
+Let's take this custom person error model as an example. 
+Every custom error model must should extend the `Error` interface and have a `JsonApiXError` annotation. 
+
+```kotlin
+@Serializable
+@JsonApiXError(type = "person")
+data class PersonError(
+    val code: String,
+    val title: String
+) : Error
+```
+
+In this example, the annotation processor will automatically make the error type of a `Person` class to be a `PersonError` and use it as a type when deserializing errors array. Developer needs to make sure that the `type` parameter value in `JsonApiXError` matches the one in the `JsonApiX` above the original model.
```

Comment:
Looks good, but I would add a  small code example with the new generic extension `asJsonXHttpException<PersonError>()`

Reply:
Agreed 👍